readme.md

```diff
@@ -7,51 +7,27 @@ InVEST models, and to be extensible to future models or common auxilary
 workflows of an InVEST user.
 
 ## To develop and launch this Application
-
-* `npm install`  
-* bind to an `invest` executeable (see package.json "invest" for a compatible version)
-
-In production, the invest exe comes from prebuilt binaries that are an artifact of the `invest` build process.  
-
-For development, duplicate the production setup by fetching prebuilt binaries:  
-`npm run fetch-invest`.  
-Or build your own local invest binaries using invest's `make binaries` and then
-copy the entire `dist/invest/` dir to this repo's `build/invest/`.  
-
+* `npm install`
+* `npm run fetch-invest`
+	+ fetches prebuilt invest binaries (see package.json invest property)
+	+ Alternatively, build your own local invest binaries:
+		* use invest's `make binaries`, then
+		* `cp -r invest/dist/invest/ invest-workbench/build/invest/`
+* `npm run dev` (this process stays live, do it in a separate shell)
 * `npm start`
 
-_Optional: using the react-devtools extension_  
-
-Before running `npm start`, in a second shell launch `react-devtools` (or `npx react-devtools` if you don't have a global install). Then run `npm start`.  
-The electron app should communicate with the
-standalone react-devtools window. There's an oustanding issue in electron
-preventing react-devtools from integrating with the electron browser's devtools window. https://github.com/electron/electron/issues/23662
-
-## To package this app for distribution  
-
-`npm run fetch-invest` - downloads invest binaries and updates the sampledata_registry.json.
+#### To run javascript outside the electron runtime,
+but with the same ECMAscript features and babel configurations:  

```

Comment:
Is this the desired markdown syntax you wanted? Just checking.

Reply:
more/less, yeah. I think it looks okay.